be/src/runtime/stream_load/stream_load_executor.cpp

```diff
@@ -61,12 +61,10 @@ Status StreamLoadExecutor::execute_plan_fragment(StreamLoadContext* ctx) {
                 ctx->number_filtered_rows = executor->runtime_state()->num_rows_load_filtered();
                 ctx->number_unselected_rows = executor->runtime_state()->num_rows_load_unselected();
 
-                int64_t num_selected_rows =
-                    ctx->number_total_rows - ctx->number_unselected_rows;
-                if ((0.0 + ctx->number_filtered_rows) / num_selected_rows > ctx->max_filter_ratio) {
-                    status = Status::InternalError("too many filtered rows");
-                }
-                else if(ctx->number_loaded_rows == 0){
+                int64_t num_selected_rows = ctx->number_total_rows - ctx->number_unselected_rows;
+                if ((double)ctx->number_filtered_rows / num_selected_rows > ctx->max_filter_ratio) {
+                    status = Status::InternalError("too many rows are filtered out");
```

Comment:
Do not change this error msg, leave it as `"too many filtered rows"`. It is long history, and some user may depends on this certain msg.

Reply:
Ok